Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM       ubuntu:18.04
```

Comment:
I think it should be documented that the systemd c libraries must match in the log shipper image and the host system.

Reply:
oh, that's kinda aweful :/

Reply:
mmm, ok, in that case, I'll make this into a multi-stage build example that "works for me"